gr-blocks/lib/pack_k_bits_bb_impl.h

```diff
@@ -32,11 +32,10 @@ namespace blocks {
 class pack_k_bits_bb_impl : public pack_k_bits_bb
 {
 private:
-    kernel::pack_k_bits* d_pack;
+    const kernel::pack_k_bits d_pack;
```

Comment:
I like! having a private variable be a locally allocated and deallocated and pointed to object makes no sense. Use C++'s own lifetime management :1

Reply:
:100: